test/integration/crcsuite/prepare.go

```diff
@@ -0,0 +1,73 @@
+// +build integration
+
+package crcsuite
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"os/user"
+	"path/filepath"
+
+	"github.com/code-ready/crc/pkg/download"
+)
+
+// Download bundle for testing
+func DownloadBundle(bundleLocation string, bundleDestination string) (string, error) {
+
+	if bundleLocation[:4] != "http" {
```

Comment:
This I don't like but we need to expand the use of `Download` function to support `file` URI also, which is by default is not present.

Reply:
I agree it's not nice. It's a temp solution as you write. I made a note.